src/platform/nxp/k32w/k32w1/BUILD.gn

```diff
@@ -52,7 +52,6 @@ static_library("nxp_platform") {
     "K32W1Config.h",
     "KeyValueStoreManagerImpl.cpp",
     "KeyValueStoreManagerImpl.h",
-    "Logging.cpp",
```

Comment:
Is it added to some "logging" target?

Reply:
It's here:
https://github.com/project-chip/connectedhomeip/blob/b0a105817784e5bd48ad3674cefaf64294936480/src/platform/nxp/BUILD.gn#L23-L34